namui-animation-editor/src/graph_editor/graph_window/mod.rs

```diff
@@ -0,0 +1,332 @@
+use namui::{
+    animation::{KeyframeGraph, Layer},
+    math::num::{FromPrimitive, ToPrimitive},
+    prelude::*,
+    types::{Angle, OneZero, Percent, Px, Time, TimePerPx},
+};
+use namui_prebuilt::{
+    table::{fixed, ratio, vertical},
+    *,
+};
+use std::{
+    fmt::Debug,
+    ops::{Div, Range},
+    sync::Arc,
+};
+mod render;
+use crate::{
+    time_ruler,
+    types::{ActionTicket, AnimationHistory},
+};
+mod update;
```

Comment:
```suggestion
mod render;
mod update;

use crate::{
    time_ruler,
    types::{ActionTicket, AnimationHistory},
};
use namui::{
    animation::{KeyframeGraph, Layer},
    math::num::{FromPrimitive, ToPrimitive},
    prelude::*,
    types::{Angle, OneZero, Percent, Px, Time, TimePerPx},
};
use namui_prebuilt::{
    table::{fixed, ratio, vertical},
    *,
};
use std::{
    fmt::Debug,
    ops::{Div, Range},
    sync::Arc,
};
```
This is a result of gouping linting imports